helm/hpcc/templates/_helpers.tpl

```diff
@@ -535,7 +535,11 @@ securityContext:
   runAsNonRoot: true
   runAsUser: 10000
   runAsGroup: 10001
+{{- if .root.Values.global.privileged }}
+  readOnlyRootFilesystem: false
```

Comment:
You can probably move this up into the existing if.
@richardkchapman ?

Reply:
@wangkx - Agree, looks fine, but would be better to move into existing if privilege block a few lines aboe

Reply:
Ok, I will move them.